.github/workflows/integration-tests.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Checkout the repository
         uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # renovate: tag=v3.5.2
       - name: Setup go
-        uses: actions/setup-go@4d34df0c2316fe8122ab82dc22947d607c0c91f9 # renovate: tag=v4.0.0
+        uses: actions/setup-go@fac708d6674e30b6ba41289acaab6d4b75aa0753 # v4.0.1
```

Comment:
I just realized, is it expected that renovate removes the renovate tag? :thinking: 
cc @chancez

Reply:
It's not "removed", I think the structure they document is just different than what they actually use. I noticed this in Cilium too. Should be fine.